src/screens/OtherProfile/index.js

```diff
@@ -642,9 +637,12 @@ const OtherProfile = () => {
           extraData={[feeds]}
           data={handleDataFeed()}
           onScroll={handleScroll}
-          onEndReach={__handleOnEndReached}
           onRefresh={handleRefresh}
           refreshing={loading}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onMomentumScrollEnd` event handler now includes two functions: `onWillSendViewPostTime` and `fetchNextFeeds`. Ensure that these functions are not blocking (i.e., they don't contain any synchronous operations that might block the main thread), as this could negatively impact scroll performance. If they do contain such operations, consider offloading them to a Web Worker or using some other form of asynchronous processing.


<!-- This is an auto-generated comment by OSS CodeRabbit -->